tests/playwright/sanity/lighthouse.test.ts

```diff
@@ -0,0 +1,20 @@
+import { playAudit } from 'playwright-lighthouse';
+import { test } from '@playwright/test';
+import config from 'lighthouse/lighthouse-core/config/desktop-config';
+
+test.describe( 'audit example', () => {
+	test( 'open browser', async ( { page } ) => {
+		await page.goto( 'http://localhost:8888/law-firm-about/' );
```

Comment:
One question about this: how can we test this for widgets?

Could we create a new page, add our content, publish the page, and test the page?
And remove the page afterwards?

I will approve this PR, once I know how we can use it.

Reply:
@dmaziar

Reply:
yes, this sounds good. I'll extend this test with widget.

Reply:
Updated. Added create a page, load template from element regression, publish and then do lighthouse check. 
Ideas, comments are welcome!